src/webgpu/api/validation/resource_usages/textureUsageInRender.spec.ts

```diff
@@ -173,3 +175,97 @@ g.test('readwrite_upon_aspects')
       encoder.finish();
     }, !success);
   });
+
+g.test('binding_types_combination')
+  .params([
```

Comment:
Hmm. It might be not important right now. In addition, the first type can be "render-target" too and I suppose the expected test result should be a failure. I will add that test later (I have already added a TODO)